### PR TITLE
fix: improve docs

### PR DESCRIPTION
**Pull request by daniel-sanche**
**Problem**: 
The generated docs on [googleapis.dev](https://googleapis.dev/python/logging/latest/index.html) are miseading. They advertise two versions of the API, when we actually only publish v2. It also makes it seem like the autogenerated library is the newer version and the handwritten layer is deprecated

**This PR**:
- removed the autogenerated layer from the generated documentation at [googleapis.dev](https://googleapis.dev/python/logging/latest/index.html)
- removed the misleading `v1` version from [googleapis.dev](https://googleapis.dev/python/logging/latest/index.html)
- improved the code samples on the README to use the hand-written layer instead of autogenerated code
- changed branding in the docs from Stackdriver Logging to Cloud Logging and Container Engine to Kubernetes Engine
- got the Kokoro docs-presubmit check to turn green

This should greatly reduce customer confusion

Fixes #https://github.com/googleapis/python-logging/issues/76 🦕

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Changelog
+Changelog
```
Comment (busunkim96) [inline CHANGELOG.md:1]: This is a markdown file - was the change to the header intentional?

File: README.rst [hunk 2]
```diff
@@ -77,57 +79,40 @@ Windows
...
-    """
-    e = logging_v2.types.LogEntry(
-        log_name="projects/[PROJECT_ID]/logs/test-logging", # optional
-        resource=resource, # optional
-        text_payload="this is a log statement")
-
-    entries = [e]
-    response = client.write_log_entries(entries)
+    # Imports the Cloud Logging client library
+    import google.cloud.logging
```
Comment (busunkim96) [inline README.rst:87]: I prefer not to have any codeblocks in the README. There's no good way to test them and it seems like a matter of time until they fall out of date.

Comment (Daniel-Sanche) [reply README.rst:87]: Good point. My intention was to remove the outdated samples, but I'm fine with removing them altogether. Done

File: CHANGELOG.md (hunk 1 above)
Comment (Daniel-Sanche) [reply CHANGELOG.md:1]: Good question, I don't think it was intentional? I removed it and it works fine

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**